app/src/pages/__tests__/Home.test.js

```diff
@@ -47,20 +47,32 @@ const mockForms = [
 
 jest.mock('../../database/crud/crud-forms');
 jest.mock('../../store/forms');
+/*
+jest.mock('../../lib', () => ({
```

Comment:
please remove any comments if you don't use it @ifirmawan

Reply:
on it mas